libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -1,172 +1,204 @@
-        <AgGridColumn
-          headerName={t('Type')}
-          field="type"
-          valueFormatter={({ value }: ValueFormatterParams) =>
-            value ? AccountTypeMapping[value as AccountType] : '-'
-          }
-        />
-        <AgGridColumn
-          headerName={t('Market')}
-          field="market.tradableInstrument.instrument.name"
-          valueFormatter="value || '—'"
-        />
-        <AgGridColumn
-          headerName={t('Balance')}
-          field="balance"
-          cellRenderer="PriceCell"
-          type="rightAligned"
-          valueFormatter={({
-            value,
-            data,
-          }: AccountsTableValueFormatterParams) =>
-            addDecimalsFormatNumber(value, data.asset.decimals)
-          }
-        />
-      </AgGrid>
+      <>
+        <AgGrid
+          style={{ width: '100%', height: '100%' }}
+          overlayNoRowsTemplate={t('No accounts')}
+          getRowId={({ data }: { data: AccountFields }) => data.asset.id}
+          ref={ref}
+          rowHeight={34}
+          tooltipShowDelay={500}
+          defaultColDef={{
+            flex: 1,
+            resizable: true,
+            tooltipComponent: TooltipCellComponent,
+            sortable: true,
+          }}
+          {...props}
+        >
+          <AgGridColumn
+            headerName={t('Asset')}
+            field="asset.symbol"
+            headerTooltip={t(
+              'Asset is the collateral that is deposited into the Vega protocol.'
+            )}
+            cellRenderer={({
+              value,
+            }: VegaValueFormatterParams<AccountFields, 'asset.symbol'>) => {
+              return (
+                <ButtonLink
+                  data-testid="deposit"
+                  onClick={() => {
+                    onClickAsset(value);
+                  }}
+                >
+                  {value}
+                </ButtonLink>
+              );
+            }}
+            maxWidth={300}
+          />
+          <AgGridColumn
+            headerName={t('Deposited')}
+            field="deposited"
+            valueFormatter={({
+              value,
+              data,
+            }: VegaValueFormatterParams<AccountFields, 'deposited'>) =>
+              data &&
+              data.asset &&
+              addDecimalsFormatNumber(value, data.asset.decimals)
+            }
+            maxWidth={300}
+          />
+          <AgGridColumn
+            headerName={t('Used')}
+            field="used"
+            flex={2}
+            maxWidth={500}
+            headerComponentParams={progressBarHeaderComponentParams}
+            cellRendererSelector={progressBarCellRendererSelector}
+            valueFormatter={progressBarValueFormatter}
+          />
+          <AgGridColumn
+            headerName=""
+            field="breakdown"
+            maxWidth={150}
+            cellRenderer={({
+              value,
+            }: VegaICellRendererParams<AccountFields, 'breakdown'>) => {
```

Comment:
suggestion (non blocking): `VegaICellRendererParams` (and elsewhere in this file.